src/permutat.cc

```diff
@@ -860,6 +886,15 @@ static Obj PowPerm(Obj opL, Obj opR)
     degC = degL < degR ? degR : degL;
     cnj = NEW_PERM<Res>( degC );
 
+    // Handle trivial cases
+    if (degL == 0) {
+        return IdentityPerm;
```

Comment:
This is fine, but I wonder: above in `PowPermInt`, you returned `opL` in a comparable situation; I'd expect the code to do the same in both cases (I don't care which of the two it does, though)